aiven/client/cli.py

```diff
@@ -1136,6 +1136,140 @@ def service_acl_list(self):
 
+    @arg.project
+    @arg.service_name
+    def service_connector_available(self):
+        """List available Kafka connectors"""
+        project_name = self.get_project()
+
+        response = self.client.get_available_kafka_connectors(project_name, self.args.name)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    def service_connector_list(self):
+        """List Kafka connectors"""
+        project_name = self.get_project()
+
+        response = self.client.list_kafka_connectors(project_name, self.args.name)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    @arg("connector", help="Connector name")
+    def service_connector_status(self):
+        """Get Kafka connector status"""
+        project_name = self.get_project()
+
+        response = self.client.get_kafka_connector_status(project_name, self.args.name, self.args.connector)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    @arg("connector", help="Connector name")
+    def service_connector_configuration(self):
+        """Get Kafka connector configuration"""
+        project_name = self.get_project()
+
+        response = self.client.get_kafka_connector_configuration(project_name, self.args.name, self.args.connector)
+
+        self.print_response(response)
+
+    def get_connector_config(self, path_or_string):
+        config = path_or_string
+        if os.path.isfile(config):
+            with open(config, 'r') as config_file:
+                config = config_file.read()
+        return config
+
+    @arg.project
+    @arg.service_name
+    @arg("connector_config", help="Connector configuration as a json file or string")
+    def service_connector_create(self):
+        """Create a Kafka connector"""
+        project_name = self.get_project()
+
+        config = self.get_connector_config(self.args.connector_config)
+        response = self.client.create_kafka_connector(project_name, self.args.name, config)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    @arg("connector", help="Connector name")
+    @arg("connector_config", help="Connector configuration as a json file or string")
+    def service_connector_update(self):
+        """Update a Kafka connector"""
+        project_name = self.get_project()
+
+        config = self.get_connector_config(self.args.connector_config)
+        response = self.client.update_kafka_connector(project_name, self.args.name, self.args.connector, config)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    @arg("connector", help="Connector name")
+    def service_connector_delete(self):
+        """Delete a Kafka connector"""
+        project_name = self.get_project()
+
+        response = self.client.delete_kafka_connector(project_name, self.args.name, self.args.connector)
+
+        self.print_response(response["message"])
```

Comment:
Does `self.print_response` actually work for simple strings? I thought it requires `dict` or list of `dict`s.

Reply:
It does work but I will remove the printing to be more consistent with the rest of the code.